roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: install helm
+  become: yes
+  become_user: root
+  command: "curl -s {{ helm_url }} | bash"
+  args:
+    chdir: "{{ helm_folder }}"l
+    creates: "{{ helm_bin }}"
+  environment: "{{ proxy_env }}"
+  tags:
+    - helm
+
+- name: install helm packages
+  become: yes
+  become_user: root
+  run_once: True
+  when: "helm_{{ item.name }}_enabled | bool"
```

Comment:
wondering if it makes sense to use this pattern here? We could just provide a list of packages to install. By default that could include Core packages or be empty and if populated would run the install. Then it wouldn't be necessary to provide a package list and a toggle variable. I'm not sure if that makes it better or worse though.

We don't provide a method for uninstalling a package either here.